cmd/termination/termination.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package termination
```

Comment:
Any reason for this package being in `cmd/*` instead of `pkg/*` ?
Usually we make the assumption that any package `cmd/foo` produce a binary called `foo`, which is not the case for this one (as there is no main).

Reply:
This is basically a library only for command binaries so I was trying to colocate it here, but it seems like that's a little unclear. I'll move it.